cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1865,3 +1865,31 @@ if(NOT ROOT_HAVE_CXX_ATOMICS_WITHOUT_LIB)
     set(ROOT_ATOMIC_LIBS ${ROOT_ATOMIC_LIB})
   endif()
 endif()
+
+#------------------------------------------------------------------------------------
+# Check if the pyspark package is installed on the system.
+# The FindPySpark.cmake module will first look for pyspark dependencies:
+# 1. It will look for Java in the system. Minimum version 1.8
+# 2. It will try to `import py4j` using PYTHON_EXECUTABLE_Development_Main
```

Comment:
I don't think this needs to be called out here; to me that's an implementation detail of `FindPySpark.cmake`.